shacl12-test-suite/tests/core/node/maxListLength-001.ttl

```diff
@@ -0,0 +1,60 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix ex: <http://datashapes.org/sh/tests/core/node/maxListLength-001.test#> .
+@prefix mf: <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sht: <http://www.w3.org/ns/shacl-test#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ex:ListShape
+  rdf:type sh:NodeShape ;
+  sh:maxListLength 2 ;
+  # Satisfies all constraints
+  sh:targetNode ( 1 2 ), rdf:nil ;
```

Comment:
So my initial comment (that I later deleted) turned out to be correct. This doesn't work. When you use a blank node such as ( 1 2) as targetNode, it will be evaluated as a node expression, which means it will try to walk the rdf:List and then use each list member as target node. To work around this for the test cases, I think all lists need to be URI nodes.